### DISCUSSION TITLE
GJK algorithm

### DISCUSSION THREAD

**Discussion by Simone2021space** [sympy/sympy#22668] [Q&A]
Hi,
did anyone implement the GJK algorithm in pure symbolic form with sympy?

If not, it is possible to do such implementation?

**Comment by oscarbenjamin** [comment]
Do you mean the Gilbert–Johnson–Keerthi distance algorithm?

https://en.wikipedia.org/wiki/Gilbert%E2%80%93Johnson%E2%80%93Keerthi_distance_algorithm

**Reply by Simone2021space** [reply]
Yes
